addons/sale_timesheet/tests/test_project_overview.py

```diff
@@ -78,16 +88,6 @@ def test_project_overview_by_project(self):
             'product_uom_id': self.product_expense.uom_id.id,
         })
 
-        other_revenues = self.env['account.analytic.line'].create({
-           'name': 'pther revenues on project_so',
-           'account_id': project_so.analytic_account_id.id,
-           'employee_id': self.employee_user.id,
-           'unit_amount': 1,
-           'amount': self.product_expense.list_price,
-           'product_id': self.product_expense.id,
-           'product_uom_id': self.product_expense.uom_id.id,
-        })
-        
```

Comment:
Removing test data in stable version means we already know there will be a change in the flow that will surely impact some odoo databases.

I do think during a bug fix we should focus on fixing the issue and not improving things (see [stable policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean)) and so removing a test which works and was validated by PO/guru is very risky.

I do also think you set a maximum effort to fix this issue and here I want to recognize the hard work you had, this is not trivial and this functionality lacks of specification which makes things harder for you to fix and harder for me to review.